openapi/openapi.yaml

```
@@ -0,0 +1,566 @@
+openapi: 3.0.3
+info:
+  title: Stregsystem
+  description: |-
+    This is the API implemented in the Stregsystem Django App.
+    The API describes ways to interact with Stregsystem without using the web-interface.
+    Existing client software utilizing the API include Stregsystem-CLI (STS) and Fappen (F-Club Web App).
+    
+    Disclaimer - The implementation is not generated using this specification, therefore they can get out of sync if changes are made directly to the codebase without updating the OpenAPI specification file accordingly.
+  version: "1.1"
+externalDocs:
+  description: Find out more about Stregsystemet at GitHub.
+  url: https://github.com/f-klubben/stregsystemet/
+servers:
+  - url: https://stregsystem.fklub.dk
+tags:
+  - name: Member
+    description: Related to the individual member page.
+  - name: Products
+    description: Related to the products.
+  - name: Sale
+    description: Related to performing a sale.
+paths:
+  /api/member:
+    get:
+      tags:
+        - Member
+      summary: Get member info
+      description: Gets a member's balance, username, active-status and name.
+      operationId: get_member_info
+      parameters:
+        - $ref: '#/components/parameters/member_id'
+      responses:
+        '200':
+          $ref: '#/components/responses/MemberFound'
+        '400':
+          $ref: '#/components/responses/MemberIdParameter_BadResponse'
+  /api/member/balance:
+    get:
+      tags:
+        - Member
+      summary: Get member balance
+      description: Gets a member's balance.
+      operationId: get_member_balance
+      parameters:
+        - $ref: '#/components/parameters/member_id'
+      responses:
+        '200':
+          $ref: '#/components/responses/MemberFound_Balance'
+        '400':
+          $ref: '#/components/responses/MemberIdParameter_BadResponse'
+  /api/member/active:
+    get:
+      tags:
+        - Member
+      summary: Get member active-status
+      description: Gets whether a member is active.
+      operationId: get_member_active
+      parameters:
+        - $ref: '#/components/parameters/member_id'
+      responses:
+        '200':
+          $ref: '#/components/responses/MemberFound_Active'
+        '400':
+          $ref: '#/components/responses/MemberIdParameter_BadResponse'
+  /api/member/sales:
+    get:
+      tags:
+        - Member
+      summary: Get member sales
+      description: Gets a list of a member's sales.
```

Purchases might be a better term? I know this is not the term used by the API, but it is clearer. Members don't have sales; they don't sell stuff.
```suggestion
      description: Gets a list of a member's purchases.
```

---

Agree, I've "set mig blind" on it, makes good sense!

---

I think the original reasoning for the naming of the API was that it was oriented for use by FIT personnel. But the new API is targeting fembers, so it makes sense to change it